app/controllers/api/account.php

```diff
@@ -3945,7 +3945,7 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
 
 App::delete('/v1/account/mfa/authenticators/:type')
     ->desc('Delete Authenticator')
-    ->groups(['api', 'account'])
+    ->groups(['api', 'account', 'mfaProtected'])
```

Comment:
Should there be a corresponding hook for this group?

Reply:
It already exists: https://github.com/appwrite/appwrite/blob/c87a7053b4f0160ea8de2e4054f8f3cc503772de/app/controllers/shared/api/auth.php#L13-L30